packit/api.py

```diff
@@ -216,7 +221,7 @@ def update_dist_git(
             trailers = (
                 [
                     (
-                        "From-source-git-commit",
+                        FROM_SOURCE_GIT_COMMIT,
```

Comment:
I agree with you that from readability PoV, this is less readable. One needs to find out the content of the constant to fully understand the code. On the other hand, I think this is more maintainable based on the reasons above.

Reply:
Okay, let's do constants then. One more question: Would using a name like `FROM_SOURCE_GIT_TOKEN` work better? In the context of Git-trailers, "From-source-git-commit" is called a "token".

Reply:
Yup, that sounds more accurate and clear to me.

Reply:
Updated.